build/build_release.sh

```diff
@@ -279,9 +280,21 @@ if [ -z $SKIP_BUILD ]; then
   ant developer -Dproject.version=$tag
 
   popd > /dev/null
+else
+   echo "Skipping mvn clean install $MAVEN_FLAGS -DskipTests -P release"
+fi
 
+if [ -z $SKIP_COMMUNITY ]; then
+   pushd community > /dev/null
+   set +e
+   mvn clean install -P communityRelease -DskipTests $MAVEN_FLAGS || true
+   set -e
+   popd > /dev/null
+else
+   echo "Skipping mvn clean install -P communityRelease -DskipTests"
 fi
 
+
 mvn assembly:attached $MAVEN_FLAGS
```

Comment:
@tbarsballe this line was hanging out on its own, unclear if it wants to use communityRelease profile

Reply:
That looks like its supposed to be top-level, so I expect not, since `mvn assembly:attached -PcommunityRelease` doesn't actually seem to do anything different than `mvn assembly:attached` if you're not in the community dir